well-known-bots.json

```diff
@@ -7271,6 +7957,7 @@
     ],
     "pattern": "[cC]laude(?:[bB]ot|-[Ww]eb)",
     "addition_date": "2024/04/19",
+    "verification": [],
```

Comment:
No verification on claudebot? I don't think the AI stuff was on our list but we might want to consider supporting verification for them since it's a big use case right now

Reply:
Anthropic don't document any way of verifying their bots. OpenAI do, but its IP verification only which is coming next. This PR is only adding dns validation.

Reply:
I've added OpenAI to your list in the design doc